lib/Model/Message.php

```diff
@@ -161,6 +161,16 @@ public function isReplyable(): bool {
 			\in_array($this->getActorType(), [Attendee::ACTOR_USERS, Attendee::ACTOR_GUESTS]);
 	}
 
+	/**
+	 * Specifies whether a message can be replied private to
+	 */
+	public function isPrivateReplyable(): bool {
+		return $this->getMessageType() !== 'system' &&
+			$this->getMessageType() !== 'command' &&
+			$this->getRoom()->getType() !== Room::ONE_TO_ONE_CALL &&
+			\in_array($this->getActorType(), [Attendee::ACTOR_USERS, Attendee::ACTOR_GUESTS]);
```

Comment:
Should only work for `Attendee::ACTOR_USERS` and also only when the current user is `Attendee::ACTOR_USERS` because otherwise they can't have a private chat at all.

Reply:
also same as with delete, I would do this check in the UI and not output it as a flag.